clients/docs/ccloud.rst

```diff
@@ -0,0 +1,229 @@
+.. _client-examples-ccloud:
+
+Confluent Cloud CLI
+-------------------
+
+In this tutorial, you will run a |confluent-cli| client application that
+produces messages to and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  Local install of :ref:`Confluent Cloud CLI <ccloud-install-cli>` v1.13.0 or later.
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for |confluent-cli|.
+
+   .. code-block:: bash
+
+      cd clients/cloud/ccloud/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+----------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Records" section or before it?
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      ccloud kafka topic create test1
+
+#. Run the :ref:`Confluent Cloud CLI producer <ccloud_kafka_topic_produce>`,
+   writing messages to topic ``test1``, passing in additional arguments:
+
+   -  ``--parse-key --delimiter ,``: pass key and value, separated by a comma
+
+   .. code-block:: bash
+
+      ccloud kafka topic produce test1 --parse-key --delimiter ,
+
+.. Is there any output the user should verfiy when running the previous command or expected output they should see?
+
+#. Type a few messages, using a ``,`` as the separator between the message key
+   and value:
+
+   .. code-block:: bash
+
+      alice,{"count":0}
+      alice,{"count":1}
+      alice,{"count":2}
+
+#. When you are done, press ``Ctrl-C``.
+
+.. Is there a source code link for this one? I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+.. The below step orginally said "Confluent Confluent CLI Consumer" originally
+   I changed it to Confluent Cloud CLI consumer as I think that is what was meant?
+
```

Comment:
@ybyzek The below step originally said "Confluent Confluent CLI Consumer"
   I changed it to "Confluent Cloud CLI consumer" as I think that is what was meant?

Reply:
Good catch @tikimims, thank you

Reply:
Great!